aten/src/ATen/native/cuda/BatchLinearAlgebraLib.cu

```diff
@@ -162,8 +164,8 @@ inline static void _apply_svd_lib_gesvdj(const Tensor& self, Tensor& U, Tensor&
   auto U_stride = matrixStride(U);
   auto S_stride = S.size(-1);
   auto VT_stride = matrixStride(VT);
-  auto batchsize = batchCount(self);
 
+  int batchsize = cuda_int_cast(batchCount(self), "batch size");
```

Comment:
This is ok if we are sure it will never overflow but I think when working with tensor dimensions we should default to int64_t unless there's a specific reason not to such as performance or space or API requirement.